docs/user-guide/clusters.md

```diff
@@ -0,0 +1,211 @@
+
+### Edit a Node
+
+`Edit node` allows you to directly edit any node. It will open the editor which contains all the configuration settings in which the default format is YAML. You can edit multiple objects, although changes are applied one at a time.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/clusters/edit-yaml-node.jpg)
+
+* You can edit or modify the parameters or values of a node by clicking `Edit node`.
+* Click `Review Changes` to compare the changes in the YAML file.
+* Click `Update Node`.
+
+
+### Delete a Node
+
+* Search a node by name or label in the search bar.
+* On the node, click the ellipsis button and then click `Delete`.
+
+The node will be deleted from the cluster.
+
+
+## Access Cluster via Terminal for Troubleshooting
+
+User with [Super admins](https://docs.devtron.ai/global-configurations/authorization/user-access#assign-super-admin-permissions) access can now troubleshoot cluster issues by accessing the cluster terminal from Devtron. You can select an image from the list that has all CLI utilities like kubectl, helm, netshoot etc. or can use a custom image, which is publicly available.
+
+To troubleshoot a cluster or a specific node in a cluster, click the terminal symbol on the right side.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/clusters/cluster-terminal-access.jpg)
+
+* You will see the user-defined name for the cluster in Devtron. E.g. `default-cluster`.
+* In the `Nodes` field, select the node from the drop-down list you want to troubleshoot. E.g. `demo-new`.
+* Select the namespace from the drop-down list which you have added in the [Environment](https://docs.devtron.ai/global-configurations/cluster-and-environments#add-environment) section.
+* Select the image from the drop-down list which includes all CLI utilities or you can use a custom image, which is publicly available.
+* Select the terminal shell from the drop-down list (e.g. `sh`, `bash`) to troubleshoot a node.
+
+
+### One of the Use Cases: Debug Pod 
+
+You can also create pod for debugging which will connect to pod terminal. To find out why the particular pod is not running, you can check `Pod Events` and `Pod Manifest` for details.
+
+* Check the current state of the Pod and recent events with the following command:
+
+```bash
+kubectl get pods
+```
+
+* To know more information about each of these pods and to debug a pod depending on the state of the pods, run the following command:
+
+```bash
+kubectl describe pod <podname>
+```
+
+  * If the pod status shows `Running` and its taking to time to load as `Successful`,  it may be there is an error in your pod description (e.g. mypod.yaml file), and that error is silently ignored when you created the pod.
+  * If the pod status shows `Pending`, it means that it can not be scheduled onto a node. Generally this is because there are insufficient resources of one type or another that prevent scheduling.
```

Comment:
It can be because of taints on the node too, so to create a cluster terminal node without any taints is recommended
althought default taints on spot nodes of azure/gcp etc. are supported

Reply:
Done